SpotifyUserLogin/src/main/java/api/SpotifyInteractor.java

```diff
@@ -76,7 +76,7 @@ public void authorizationCodeUri() {
         // Build the request
         final AuthorizationCodeUriRequest authorizationCodeUriRequest = this.api.authorizationCodeUri()
                 .scope(APPLICATION_SCOPE)
-                .show_dialog(false)
+                .show_dialog(true)
```

Comment:
Sorry I meant is this still necessary?

Reply:
It doesn't matter. Feel free to change it if it's inconvenient. It just happened to be convenient at the time